sdks/python/apache_beam/dataframe/transforms_test.py

```diff
@@ -354,6 +354,16 @@ def test_rename(self):
                   0: 2, 2: 0
               }, errors='raise'))
 
+  def test_dataframe_column_fillna_constant_as_value(self):
```

Comment:
I would place it together with other fillna tests: https://github.com/apache/beam/blob/deb4b4ada3a414205a87de0dcad745c95f45a2e5/sdks/python/apache_beam/dataframe/frames_test.py#L987

Reply:
I have tried to put in that file like this [commit ](https://github.com/apache/beam/pull/32594/commits/62bfbfb56e6128e5f5ccdae2b95cc4353b4f1ede#diff-c4fc5ef3c0269d39bcfcb9021c0fe2778d33a4052af78b36fa5150309469affbR1012-R1018), but seem that this function is not triggered, that trigger different flow. And the test pass even with the old version of code

https://github.com/apache/beam/blob/deb4b4ada3a414205a87de0dcad745c95f45a2e5/sdks/python/apache_beam/dataframe/transforms.py#L362

Reply:
Please help to advise

Reply:
Sorry for the late reply. 

The bug only triggered in the transform of DeferredDataFrame, so you have to put that in a pipeline so that when the pipeline runs the corresponding transform code path will run. 

Could you put a test like the following into frames_test.py?

```python
import numpy as np

import apache_beam as beam
from apache_beam.dataframe import convert
from apache_beam.testing.util import assert_that
from apache_beam.testing.util import equal_to

with beam.Pipeline(None) as p:
  pcoll = (p | beam.Create([1.0, np.nan, -1.0]) | beam.Select(x=lambda x: x))
  df = convert.to_dataframe(pcoll)
  df_new = df['x'].fillna(0)
  assert_that(
    convert.to_pcollection(df_new),
    equal_to([1.0, 0.0, -1.0]))
```

Reply:
Thank you @shunping for your suggestion. That works as well